### PR TITLE
Trending shapes @ homepage

### PR DESCRIPTION
**Pull request by nirmalkc**

### CODE REVIEW

File: components/core/Landing.js [hunk 1]
```diff
@@ -425,6 +581,82 @@ const FileSupportCardItem = styled.div`
 
 const Landing = ({ setOpen, user, setUser }) => {
     
+  const [data, setData] = useState([]); // shapes
+  const [loading, setLoading] = useState(true); // shapes loading
+
+  useEffect(async () => {
```
Comment (atapas) [inline components/core/Landing.js:587]: @williamzhu17 Everything related to trending shapes move to a separate component. In landing, just call the component.

**Review by atapas — CHANGES_REQUESTED** [review]

File: components/core/Landing.js (hunk 1 above)
Comment (williamzhu17) [reply components/core/Landing.js:554]: @atapas All code relating to trending shapes should now be a separate component named TrendingShapes.

Comment (atapas) [reply components/core/Landing.js]: @williamzhu17 The changes are lookig good. I have made a few more refactoring and committed. Please check once. I shall merge it  soom. Resolving this comment.

**Review by atapas — APPROVED** [review]
Review comments are taken care

**Pull request merged**